### PR TITLE
[cloudbuild] Build all images in a single step.

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Build all images in a single cloudbuild step.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -8,23 +8,7 @@ steps:
...
-    env:
-      - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
-      - GIT_TAG=$_GIT_TAG
-      - EXTRA_TAG=$_PULL_BASE_REF
-      - DOCKER_BUILDX_CMD=/buildx-entrypoint
-  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221214-1b4dd4d69a'
-    entrypoint: make
-    args:
       - importer-image-push
     env:
```
Comment (alculquicondor) [inline cloudbuild.yaml:13]: we are missing the IMAGE_REGISTRY variable here.

Comment (alculquicondor) [reply cloudbuild.yaml:13]: Ah right.

Well, the thing is that importer and debug images will never be in `registry.k8s.io`, whereas the main `kueue` image will.

Having the 2 registries as different variables theoretically allows you to run steps for images located in different places.

Comment (trasc) [reply cloudbuild.yaml:13]: It's hard to follow, let's keep a single var for now.
